backend/app/tests/api/routes/test_improve_prompt.py

```
@@ -1,62 +1,62 @@
+- get_cloud_storage / load_json_from_object_store (traces)
+- start_prompt_improvement (the Celery enqueue helper) — never touch a broker
+- Session (the worker opens its own Session(engine); we redirect it at the
+  transactional db fixture, matching the doctransformer worker tests)
 
 DB is real (transactional db fixture; rolls back after each test).
 """
 
 import contextlib
 import json
+from contextlib import ExitStack
 from typing import Any, Iterator
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)